robottelo/ui/rhai.py

```diff
@@ -12,3 +12,13 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def org_selection_required(self):
+        """Message: 'Organization Selection Required' must be displayed if
+        the user tries to view Access Insights overview without selecting an
+        org"""
+
+        result = self.wait_until_element(
+            locators['insights.org_selection_msg']
+        ).text
```

Comment:
I am not sure if this function is good enough to be a separate utility method.  Can't this line be done in the test itself?